02-Reproject-Raster-In-R.Rmd

```diff
@@ -194,32 +205,34 @@ crs(DTM_hill_HARV)
 extent(DTM_hill_UTMZ18N_HARV)
 extent(DTM_hill_HARV)
 
-
 ```
 
 Notice in the output above that the CRS of `DTM_hillUTMZ18N_HARV` is now UTM. 
 However, the extent of the object is different. 
+
 Why do you think this is?  
 
+Note: When you are reprojecting a raster, you are moving it from one "grid" 
```

Comment:
What is the difference between notice1 and 2?
i think this should be a discussion to make sure we need two formats.